model/Core/Properties/requirementRational.md

```diff
@@ -0,0 +1,17 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# requirementRational
```

Comment:
Should this be `requirementRationale`? If yes, then, also the file name change is needed.